core/eth/client_test.go

```diff
@@ -154,3 +158,32 @@ func TestCallerSubscriberClient_GetERC20Balance(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, expected, result)
 }
+
+func TestCallerSubscriberClient_GetAggregatorPrice(t *testing.T) {
+	caller := new(mocks.CallerSubscriber)
```

Comment:
For my understanding, is there a consensus on when to use 'new' and when not to?  Asking also because of Yung Spooks comment on another PR about 'new'.

https://github.com/smartcontractkit/chainlink/pull/2021#discussion_r357357913

Reply:
I've been following precedence with regards to `new(mocks.*)` and mockery. I don't have a strong opinion about this vs `&mocks.CallerSubscriber{}`. I get the one way to do things dogma, just don't think the dialog is worth the hit to productivity.